src/utils.rs

```diff
@@ -367,14 +367,6 @@ macro_rules! out_of_file_lines_range {
     };
 }
 
-macro_rules! skip_out_of_file_lines_range {
-    ($self:ident, $span:expr) => {
-        if out_of_file_lines_range!($self, $span) {
-            return None;
-        }
-    };
-}
-
```

Comment:
Now this is totally replaced with `skip_out_of_file_lines_range!` which returns `RewriteError::SkipFormatting` instead of `None`

Reply:
You mean `skip_out_of_file_lines_range_err`? In a follow up PR I think it would be nice to rename `skip_out_of_file_lines_range_err` back to `skip_out_of_file_lines_range!` Now that we know we've replaced all the invocations to return `RewriteResult`. I'm not in a rush to make this change though.